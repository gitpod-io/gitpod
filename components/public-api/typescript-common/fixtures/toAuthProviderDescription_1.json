{
  "authProviderId": "ap123",
  "authProviderType": "GitHub",
  "host": "localhost",
  "verified": true,
  "icon": "unused icon",
  "description": "unused description",
  "settingsUrl": "unused",
  "ownerId": "unused",
  "organizationId": "unused"
}
