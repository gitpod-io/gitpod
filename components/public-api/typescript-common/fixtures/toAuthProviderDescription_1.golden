{
    "result": {
        "id": "ap123",
        "type": "AUTH_PROVIDER_TYPE_GITHUB",
        "host": "localhost",
        "icon": "unused icon",
        "description": "unused description"
    },
    "err": ""
}
